dataprofiler/labelers/base_data_labeler.py

```diff
@@ -788,3 +788,7 @@ def load_with_components(cls, preprocessor, model, postprocessor):
         data_labeler.set_model(model)
         data_labeler.set_postprocessor(postprocessor)
         return data_labeler
+
+    def close(self):
+        if hasattr(self.model, 'close'):
+            self.model.close()
```

Comment:
I don't think this is ever executed because I believe it is self._model. But in eithercase, can we have tests for the close functions?

Reply:
I don't think all models will need this functionality and therefore should not be in base. Hence why this is checking to begin with, it may not be necessary. Otherwise, models that don't need it will have functions that are not used / intended.

Reply:
This method works, I double checked. The reason I added it is to make sure we can close session after done with the model. I can move to char-cnn model if needed. Whether to keep this, I'm open for the discussion.

Reply:
I'm not sure this specifically needs to be here

Reply:
this's only for `TrainableDataLabeler` and `BaseTrainableModel`, which are only used for ml models

Reply:
Then shouldn't it be in the model class?

Reply:
i can move it to the model class if we're all fine with that

Reply:
A trainable model may or may not need close, hence I still don't think it would be appropriate here.